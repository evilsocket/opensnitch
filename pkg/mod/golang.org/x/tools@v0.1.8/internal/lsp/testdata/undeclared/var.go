package undeclared

func m() int {
	z, _ := 1+y, 11 //@diag("y", "compiler", "undeclared name: y", "error"),suggestedfix("y", "quickfix")
	if 100 < 90 {
		z = 1
	} else if 100 > n+2 { //@diag("n", "compiler", "undeclared name: n", "error"),suggestedfix("n", "quickfix")
		z = 4
	}
	for i < 200 { //@diag("i", "compiler", "undeclared name: i", "error"),suggestedfix("i", "quickfix")
	}
	r() //@diag("r", "compiler", "undeclared name: r", "error")
	return z
}
