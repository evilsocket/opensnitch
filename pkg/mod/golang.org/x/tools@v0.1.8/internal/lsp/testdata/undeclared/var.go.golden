-- suggestedfix_var_10_6 --
package undeclared

func m() int {
	z, _ := 1+y, 11 //@diag("y", "compiler", "undeclared name: y", "error"),suggestedfix("y", "quickfix")
	if 100 < 90 {
		z = 1
	} else if 100 > n+2 { //@diag("n", "compiler", "undeclared name: n", "error"),suggestedfix("n", "quickfix")
		z = 4
	}
	i := 
	for i < 200 { //@diag("i", "compiler", "undeclared name: i", "error"),suggestedfix("i", "quickfix")
	}
	r() //@diag("r", "compiler", "undeclared name: r", "error")
	return z
}

-- suggestedfix_var_4_12 --
package undeclared

func m() int {
	y := 
	z, _ := 1+y, 11 //@diag("y", "compiler", "undeclared name: y", "error"),suggestedfix("y", "quickfix")
	if 100 < 90 {
		z = 1
	} else if 100 > n+2 { //@diag("n", "compiler", "undeclared name: n", "error"),suggestedfix("n", "quickfix")
		z = 4
	}
	for i < 200 { //@diag("i", "compiler", "undeclared name: i", "error"),suggestedfix("i", "quickfix")
	}
	r() //@diag("r", "compiler", "undeclared name: r", "error")
	return z
}

-- suggestedfix_var_7_18 --
package undeclared

func m() int {
	z, _ := 1+y, 11 //@diag("y", "compiler", "undeclared name: y", "error"),suggestedfix("y", "quickfix")
	n := 
	if 100 < 90 {
		z = 1
	} else if 100 > n+2 { //@diag("n", "compiler", "undeclared name: n", "error"),suggestedfix("n", "quickfix")
		z = 4
	}
	for i < 200 { //@diag("i", "compiler", "undeclared name: i", "error"),suggestedfix("i", "quickfix")
	}
	r() //@diag("r", "compiler", "undeclared name: r", "error")
	return z
}

